modules/virt-features-for-storage-matrix.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-features-for-storage.adoc
+
+[id="virt-features-for-storage-matrix_{context}"]
+= Features for {VirtProductName} storage
+
+.{VirtProductName} storage feature matrix
+[cols="40%,15%,15%,15%,15%"]
+|===
+||Virtual machine live migration  |Host-assisted virtual machine disk cloning |Storage-assisted virtual machine disk cloning |Virtual machine import
+
+|{product-title} container storage: RBD block-mode volumes
+|Yes
+|Yes
+|Yes
+|(d)
```

Comment:
Per discussion, let's update the table footnotes per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables.